static/templates/about.html

```diff
@@ -766,11 +766,15 @@ <h2 class="amarante font-base">
             alt="Agnes Worthington, Boeda Strand and Lottie Woodman">
         </div>
         <div class="frame">
-          <img class="carousel-img" src="/static/images/hicksfamily.png" alt="william shelton spirit pole">
+          <img class="carousel-img" src="/static/images/william-we-ah-lup-smoking-salmon.png"
+            alt="william shelton we ah lup william-we-ah-lup-smoking-salmon">
```

Comment:
I suspect the alt text here should be something like `William Shelton We Ah Lup smoking salmon` instead?